### DISCUSSION TITLE
scATAC seq starting point

### DISCUSSION THREAD

**Discussion by incredible-maker** [stuart-lab/signac#1357] [Q&A] [answered]
Hi, I downloaded a scATAC file from GEO, and it consists of three files: barcodes.tsv.gz, matrix.mtx.gz, and peaks.bed.gz. However, the tutorial for Signac requires an h5 file and a fragment file. I want to know if there is any way to read this data by Signac even though it requires completely different input files.

![4c1d5239d024b9756a0f17333bdf8ab](https://user-images.githubusercontent.com/117089393/225581746-73c5eebc-ceff-40c8-b2e7-33d4d04876bf.png)

**Comment by timoast** [comment, marked as answer]
Signac does not require an h5 file, it only requires a matrix. The files you have can be used to construct a matrix in R (try the ReadMtx function)

Many functions in Signac do require a fragment file, so if you only have a matrix you won't be able to run all the steps shown in the website vignettes
